$MirOS: ports/www/php/extensions/patches/patch-ext_curl_interface_c,v 1.2 2008/05/02 15:33:47 tg Exp $

	Some deprecated functions have been removed from
	recent curl versions, so PHP can’t use them any
	more either… this will break scripts using them.

--- ext/curl/interface.c.orig	Thu Oct 18 17:45:53 2012
+++ ext/curl/interface.c	Thu Oct 18 17:51:45 2012
@@ -170,7 +170,6 @@ PHP_MINIT_FUNCTION(curl)
 	REGISTER_CURL_CONSTANT(CURLOPT_FTPAPPEND);
 	REGISTER_CURL_CONSTANT(CURLOPT_NETRC);
 	REGISTER_CURL_CONSTANT(CURLOPT_FOLLOWLOCATION);
-	REGISTER_CURL_CONSTANT(CURLOPT_FTPASCII);
 	REGISTER_CURL_CONSTANT(CURLOPT_PUT);
 #if CURLOPT_MUTE != 0
 	REGISTER_CURL_CONSTANT(CURLOPT_MUTE);
@@ -208,7 +207,6 @@ PHP_MINIT_FUNCTION(curl)
 	REGISTER_CURL_CONSTANT(CURLOPT_FILETIME);
 	REGISTER_CURL_CONSTANT(CURLOPT_WRITEFUNCTION);
 	REGISTER_CURL_CONSTANT(CURLOPT_READFUNCTION);
-	REGISTER_CURL_CONSTANT(CURLOPT_PASSWDFUNCTION);
 	REGISTER_CURL_CONSTANT(CURLOPT_HEADERFUNCTION);
 	REGISTER_CURL_CONSTANT(CURLOPT_MAXREDIRS);
 	REGISTER_CURL_CONSTANT(CURLOPT_MAXCONNECTS);
